File: Data Structures & Algorithms/longest-consecutive-sequence/submission-2.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        s = set()
        u = []
        res = 0
        for i in nums:
            if i not in s:
                s.add(i)
                u.append(i)
        
        for i in u:
            count = 1
            k = i
            while (k+1) in s:
                count += 1
                k+=1
            res = max(res, count)
        return res

File: Data Structures & Algorithms/house-robber/submission-2.java
class Solution {
    public int rob(int[] nums) {
        int a = 0, b = nums[0];
        int n = nums.length;

        for(int i = 1; i < n; i++) {
            int temp = b;
            b = Math.max(a + nums[i], b);
            a = temp;
        }

        return Math.max(a,b);
    }
}